superset-frontend/src/views/CRUD/welcome/SavedQueries.tsx

```diff
@@ -57,19 +64,45 @@ interface SavedQueriesProps {
   mine: Array<Query>;
 }
 
+export const CardStyles = styled.div`
+  cursor: pointer;
+  a {
+    text-decoration: none;
+  }
+  .ant-card-cover {
+    border-bottom: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
+  }
+`;
+
 const QueryData = styled.div`
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-  border-bottom: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
-  .title {
-    font-weight: ${({ theme }) => theme.typography.weights.normal};
-    color: ${({ theme }) => theme.colors.grayscale.light1};
+  svg {
+    margin-left: ${({ theme }) => theme.gridUnit * 10}px;
   }
-  .holder {
-    margin: ${({ theme }) => theme.gridUnit * 2}px;
+  .query-title {
+    padding: ${({ theme }) => theme.gridUnit * 2 + 2}px;
+    font-size: ${({ theme }) => theme.typography.sizes.l}px;
   }
 `;
+
+const ShowEmptyState = styled.div`
```

Comment:
Is there a reason not to use the <Empty> component like we're doing in other sections?

Reply:
The way the logic is set for the others  it checks  if there is thumbnail url where here we are checking if if there is a query. The way that I'm using the card is a little different from the chart and dashboard since we are depeneding on whether there is a block code of sql for the reactsyntaxhighlighe to show.